.gitignore

```diff
@@ -6,4 +6,4 @@ npm-debug.log
 # SQLite Files
 *.db
 *-wal
-*-shm
+*-shm
```

Comment:
What's the point of this edit? It seems like your editor removes trailing spaces from files (or at the very least doesn't ensure they're there), which is generally speaking not correct. Should be simple VSCode setting though.